generators/typescript/utils/commons/src/typescript-project/BundledTypescriptProject.ts

```diff
@@ -67,7 +67,7 @@ async function bundle({ platform, target, format, outdir }) {
         platform,
         target,
         format,
-        entryPoint: "./${TypescriptProject.SRC_DIRECTORY}/index.ts",
+        entryPoint: "./${this.packagePath}/index.ts",
```

Comment:
This is more a naming thing.
`packagePath` indicates to me that the entire package is moved.
I expect package.json/src/tests/etc. all to move together. 
I'd suggest renaming this to something else.